packages/quantic/force-app/main/default/lwc/quanticResultLink/quanticResultLink.js

```diff
@@ -1,9 +1,8 @@
-import {LightningElement, api} from 'lwc';
-import {
-  getHeadlessBundle,
-  getHeadlessEnginePromise,
-} from 'c/quanticHeadlessLoader';
-import {ResultUtils} from 'c/quanticUtils';
+import { getHeadlessBundle, getHeadlessEnginePromise } from 'c/quanticHeadlessLoader';
+import { ResultUtils } from 'c/quanticUtils';
+import { NavigationMixin } from 'lightning/navigation';
+import { LightningElement, api } from 'lwc';
+
 
 /** @typedef {import("coveo").Result} Result */
 /** @typedef {import("coveo").SearchEngine} SearchEngine */
```

Comment:
It would be nice to update the description of this component in line `11` in order to reflect the new capability that got added to the component.

Because this description is the one will be displayed in our documentation.

Reply:
this is good ? 

>  The `QuanticResultLink` component creates a clickable link from a result that points to the original item.
>  If the result has is of type `Salesforce` it will open the link in a new salesforce console subtab.
>  If the result is of type other than `Salesforce`, it will open the link in the browser tab.

Reply:
LGTM!